hadoop-hdds/rocks-native/src/main/java/org/apache/hadoop/hdds/utils/NativeLibraryNotLoadedException.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.hdds.utils;
+
+/**
+  Exception when native library not loaded.
+ */
+public class NativeLibraryNotLoadedException extends Exception {
+  public NativeLibraryNotLoadedException(String libraryName) {
+    super(String.format("Unable to load library %s from both " +
+            "java.library.path & resource file %s from jar.", libraryName,
+            NativeLibraryLoader.getJniLibraryFileName(libraryName)));
+  }
+}
```

Comment:
nit: new line at the end of the file
```suggestion
}

```